---
float64 d1
float64 alpha1
float64 d2
float64 alpha2
float64 diag
float64 theta
float64 cornerAngleInDeg